### ISSUE TITLE
[CI] SnapshotBasedIndexRecoveryIT#testNodeDisconnectsDoNotOverAccountRecoveredBytes timeout

### ISSUE THREAD

**Issue by cbuescher** [elastic/elasticsearch#90677]
### CI Link

https://gradle-enterprise.elastic.co/s/n2rdnntuvhw64

### Repro line

-

### Does it reproduce?

No

### Applicable branches

main

### Failure excerpt

The test suite times out but I could see the following in the jstack output at timeout time

```
  2> "TEST-SnapshotBasedIndexRecoveryIT.testNodeDisconnectsDoNotOverAccountRecoveredBytes-seed#[2725E4BEF73A48D3]" ID=33 WAITING on java.util.concurrent.CountDownLatch$Sync@79e0a85c
  2> 	at java.base@17.0.2/jdk.internal.misc.Unsafe.park(Native Method)
  2> 	- waiting on java.util.concurrent.CountDownLatch$Sync@79e0a85c
  2> 	at java.base@17.0.2/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
  2> 	at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquire(AbstractQueuedSynchronizer.java:715)
  2> 	at java.base@17.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1047)
  2> 	at java.base@17.0.2/java.util.concurrent.CountDownLatch.await(CountDownLatch.java:230)
  2> 	at app//org.elasticsearch.xpack.snapshotbasedrecoveries.recovery.SnapshotBasedIndexRecoveryIT.testNodeDisconnectsDoNotOverAccountRecoveredBytes(SnapshotBasedIndexRecoveryIT.java:1373)
```

https://gradle-enterprise.elastic.co/s/n2rdnntuvhw64/console-log/raw?task=:x-pack:plugin:snapshot-based-recoveries:internalClusterTest

Looks like the firstFileChunkSent latch hasn't been invoked yet. 
I saw this test is quite new so assigning @fcofdez who seems to be the author for a first look.

**Comment by fcofdez** [comment]
Closing this as a duplicate of https://github.com/elastic/elasticsearch/issues/90665
